pkg/fanal/secret/builtin-rules.go

```diff
@@ -142,6 +142,14 @@ var builtinRules = []Rule{
 		Regex:    MustCompile(`ghr_[0-9a-zA-Z]{76}`),
 		Keywords: []string{"ghr_"},
 	},
+	{
+		ID:       "github_pat",
```

Comment:
```suggestion
		ID:       "github-fine-grained-pat",
```

Reply:
Can you also change [github-pat](https://github.com/aquasecurity/trivy/blob/273271b462936ab1f84b8bb88f66fa71b216ccec/pkg/fanal/secret/builtin-rules.go#L113-L120) rule:
- ID: `github-pat` to `github-classic-pat`
- Title: `GitHub Personal Access Token` to `GitHub Personal Access Token (classic)`

Reply:
can you also update [github-pat](https://github.com/aquasecurity/trivy/blob/273271b462936ab1f84b8bb88f66fa71b216ccec/pkg/fanal/secret/builtin-rules.go#L113-L120):
- ID: change `github_pat` to `github-classic-pat`
- Title: Change "GitHub Personal Access Token" to "GitHub Personal Access Token (Classic)".